src/main/java/gov/nasa/pds/tools/validate/content/array/ArrayContentValidator.java

```diff
@@ -252,6 +252,16 @@ private void validatePosition(Array array, ArrayObject arrayObject, ArrayLocatio
           LOG.warn("validatePosition:Unhandled dataType {}", dataType);
           break;
       }
+    } catch (ArrayIndexOutOfBoundsException aiobe) {
+      String message = "No message specified, but probably index out of range. Report this message to have it fixed (have original exception given a message";
```

Comment:
Extreme pedant here: closing `)` to match the `(` surrounding `have original exception given a message`.

Reply:
fixed